apps/web/vibes/soul/sections/product-detail/index.tsx

```diff
@@ -137,6 +139,7 @@ export function ProductDetail<F extends Field>({
                     }
                   </Stream>
 
+                  <h2 className="sr-only">{additionalInformationLabel}</h2>
```

Comment:
🍹 Should we name this `additionalInformationTitle` instead?

Also, is "additional information" the right way to frame this? Or is there a different term like "product specifications"?

Reply:
I struggled with this, technically it can be used for anything, but yeah product specifications might fit the bill better.

Reply:
Actually, in our case, we render product specifications and warranty information, which is why I went with "additional information". I'm okay with changing to `additionaInformationTitle`.